language-server/modules/langserver-core/src/test/resources/codeaction/extract-to-constant/source/extractToConstantNegative.bal

```diff
@@ -0,0 +1,8 @@
+const int CONST1 = 10;
+const int CONST2 = 10 + 20;
+
+function testFunction() {
+    3.14;
+    int intVal = 100;
```

Comment:
Shall we support `int intVal = -100;` as well? might need to support Unary expression and do some changes in the visitor

Reply:
Done